xc/xc7/tests/soc/ibex/generate.py

```diff
@@ -65,7 +65,7 @@ def get_fusesoc_sources(root_dir, eda_yaml_path, f_log):
         eda_yaml_path=eda_yaml_path
     )
 
-    return set(
+    return list(
```

Comment:
So rather than `set(` -> `list(`, change this to `sorted(set(`

Reply:
I believe that we want to preserve the original synthesis order, so in this case, `list()` seems to be more appropriate.